packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -24,6 +25,30 @@ import { title } from './title';
 import { runRules } from './transaction-rules';
 import { batchUpdateTransactions } from './transactions';
 
+// TODO - add types for bank sync transaction data
+interface BankSyncData {
+  transactions: {
+    all: any[]; // eslint-disable-line
```

Comment:
If would be best to avoid using `any` on new types

Reply:
I've typed the relevant parts now, there will be more to be done to get this file fully-ts but it's a good start.